pkg/doc/signature/verifier/public_key_verifier_test.go

```diff
@@ -374,3 +374,38 @@ type testSignatureVerifier struct {
 func (v testSignatureVerifier) Verify(*PublicKey, []byte, []byte) error {
 	return v.verifyResult
 }
+
+func newCryptoSigner(keyType kmsapi.KeyType) (signature.Signer, error) {
+	p := mockkms.NewProviderForKMS(storage.NewMockStoreProvider(), &noop.NoLock{})
+	localKMS, err := localkms.New("local-lock://custom/master/key/", p)
```

Comment:
same comment as the previous createKMS() comment above.. 

even better yet, have the kms as an argument, to avoid creating a provider and kms on each newCryptSigner call (unless you need a new KMS each time)

Reply:
It's done for the sake of usability - it's a one-liner to create a crypto signer based on the key type.

Reply:
It's done for the sake of usability - it's a one-liner to create a crypto signer based on the key type.